modules/python/src/custom_nodes/google_genmedia/utils.py

```diff
@@ -356,92 +297,29 @@ def generate_video_from_gcsuri_image(
-
-            # return self._process_video_response(operation)
-            return process_video_response(operation)
-        except genai_errors.ClientError as e:
-            if e.code == StatusCode.RESOURCE_EXHAUSTED:
-                retries += 1
-                if retries <= retry_count:
-                    retry_wait = retry_delay
-                    print(
-                        f"API Quota/Resource Exhausted (attempt {retries}/{retry_count}) - "
-                        f"Code: {e.code.name}. Waiting {retry_wait:.1f} seconds before retry. Error: {e.details}"
-                    )
-                    time.sleep(retry_wait)
-                else:
-                    raise RuntimeError(
-                        f"API Quota/Resource Exhausted after {retries} attempts for {model} (Code: {e.code.name}). "
-                    )
-            elif e.code == StatusCode.INVALID_ARGUMENT:
-                raise ValueError(
-                    f"Invalid API argument supplied. Check your prompt and parameters. Error: {e.details}"
-                )
-            elif (
-                e.code == StatusCode.PERMISSION_DENIED
-                or e.code == StatusCode.UNAUTHENTICATED
-                or e.code == StatusCode.FORBIDDEN
-            ):
-                raise RuntimeError(
-                    f"Permission denied. Check your GCP service account permissions for Veo API. Error: {e.details}"
-                )
-            else:
-                # Catch any other ClientError that's not specifically handled (e.g., Bad Request, Conflict)
-                raise RuntimeError(
-                    f"Unexpected Veo API Client Error (Code: {e.code.name}). Error: {e.details}"
-                )
 
-        except genai_errors.ServerError as e:
-            if e.code == StatusCode.UNAVAILABLE:
-                retries += 1
-                if retries <= retry_count:
-                    retry_wait = retry_delay
-                    print(
-                        f"API Service Unavailable (attempt {retries}/{retry_count}) - "
-                        f"Code: {e.code.name}. Waiting {retry_wait:.1f} seconds before retry. Error: {e.details}"
-                    )
-                    time.sleep(retry_wait)
-                else:
-                    raise RuntimeError(
-                        f"API Service Unavailable after {retries} attempts. Giving up. Last error: {e.details}"
-                    )
-            elif e.code == StatusCode.DEADLINE_EXCEEDED:
-                raise RuntimeError(
-                    f"API request timed out (Deadline Exceeded). Error: {e.details}"
-                )
-            else:
-                # Catch any other ServerError types
-                raise RuntimeError(
-                    f"Unexpected Veo API Server Error (Code: {e.code.name}). Error: {e.details}"
-                )
+    print("Sending request to Veo API for image-to-video generation")
+    operation = client.models.generate_videos(
```

Comment:
this should be in a try except block to catch the cascading exception and then print it and exit the program.